src/ape/utils.py

```diff
@@ -31,12 +31,20 @@
     from functools import cached_property  # type: ignore
 except ImportError:
     from backports.cached_property import cached_property  # type: ignore
+"""
+Not sure what this is.
+"""
 
 try:
     from functools import singledispatchmethod  # type: ignore
 except ImportError:
     from singledispatchmethod import singledispatchmethod  # type: ignore
 
+    """
```

Comment:
Yeah don't worry about this doc here
See https://github.com/ApeWorX/ape/pull/308/files#r768188322